### PR TITLE
[15.0][ADD] helpdesk_mgmt_team_customer

### PR DESCRIPTION
**Pull request by DantePereyra**
Autofills the customer by the team selection.

### CODE REVIEW

File: helpdesk_mgmt_team_customer/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#    Copyright (C) 2020 Solvos Consultoría Informática <www.solvos.es>
```
Comment (dalonsod) [inline helpdesk_mgmt_team_customer/__manifest__.py]: Wrong year
```suggestion
#    Copyright (C) 2024 Solvos Consultoría Informática <www.solvos.es>
```

File: helpdesk_mgmt_team_customer/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Helpdesk Team Customer",
+    "summary": "Autofills the customer by the team selection",
+    "author":  "Solvos, "
```
Comment (dalonsod) [inline helpdesk_mgmt_team_customer/__manifest__.py]: Please remove OCA as author

File: helpdesk_mgmt_team_customer/i18n/es_ES.po [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Translation of Odoo Server.
```
Comment (dalonsod) [inline helpdesk_mgmt_team_customer/i18n/es_ES.po]: Proper file name for general spanish is actually `es.po`.

File: helpdesk_mgmt_team_customer/models/helpdesk_ticket.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from odoo import models, api
```
Comment (dalonsod) [inline helpdesk_mgmt_team_customer/models/helpdesk_ticket.py:3]: For every `.py` file (but `__init__.py`) add copyright comment lines

File: helpdesk_mgmt_team_customer/models/res_partner.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+from odoo import models, fields
```
Comment (dalonsod) [inline helpdesk_mgmt_team_customer/models/res_partner.py]: Remove this file, as unused.

**Review by dalonsod — CHANGES_REQUESTED** [review]
See initial comments

**Comment by DantePereyra** [comment]
Changes done.

File: helpdesk_mgmt_team_customer/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
+#    Copyright (C) 2024 Solvos Consultoría Informática <www.solvos.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Helpdesk Team Customer",
+    "summary": "Autofills the customer by the team selection",
+    "author":  "Solvos",
+    "website": "https://github.com/OCA/helpdesk",
```
Comment (dalonsod) [inline helpdesk_mgmt_team_customer/__manifest__.py]: Wrong website:
```suggestion
    "website": "https://github.com/solvosci/slv-helpdesk",
```

File: helpdesk_mgmt_team_customer/models/__init__.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+from . import res_partner
```
Comment (dalonsod) [inline helpdesk_mgmt_team_customer/models/__init__.py]: If this import is not removed, an error will raise

**Review by dalonsod — CHANGES_REQUESTED** [review]
See new comments

**Comment by DantePereyra** [comment]
Changes done and added README.rst

**Comment by dalonsod** [comment]
OK, I'll test it ASAP

**Review by dalonsod — APPROVED** [review]
Tested :+1: 

Only minor change should be made: `icon.png` file is placed in a wrong folder (it should be located inside `static/description` folder, not `static` one). Once done, I'll merge it

**Comment by DantePereyra** [comment]
Icon changed to `static/description`

**Comment by dalonsod** [comment]
Congrats @DantePereyra your first merged PR!!

**Pull request merged**